### PR TITLE
Unseen object image segmentation single frame update

### PR DESCRIPTION
**Pull request by Soofiyan**

### CODE REVIEW

File: aurmr_perception/src/aurmr_perception/grasp_detection.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from sensor_msgs.msg import Image
 from cv_bridge import CvBridge, CvBridgeError
 import sys
-sys.path.append("/home/aurmr/workspaces/soofiyan_ws/src/segnetv2_mask2_former/UIE_main/mask2former_frame/")
+sys.path.append("/home/aurmr/workspaces/uois_soofiyan_ws/src/segnetv2_mask2_former/UIE_main/mask2former_frame/")
```
Comment (markusgrotz) [inline aurmr_perception/src/aurmr_perception/grasp_detection.py:21]: Can you remove this ?

Comment (markusgrotz) [reply aurmr_perception/src/aurmr_perception/grasp_detection.py:21]: This one is not resolved yet, right?

File: aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/tools/match_masks.py [hunk 2]
```diff
@@ -15,8 +15,12 @@ def match_masks(im1, im2, mask1, mask2):
 
     mask_recs = np.zeros(shape=(np.max(mask1), np.max(mask2)))
 
-    sift = cv2.SIFT_create()
+    sift = cv2.SIFT_create(nfeatures=150, contrastThreshold=0.04, edgeThreshold=30)
     k2, d2 = sift.detectAndCompute(im2, None)
+    print()
```
Comment (markusgrotz) [inline aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/tools/match_masks.py:20]: Remove (optional)

File: aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/tools/run_network.py [hunk 3]
```diff
@@ -34,7 +34,7 @@
 import cv2
 
 # Change to reflect the path to the UOC directory
-UOC_PATH = '/home/aurmr/workspaces/thomas_ws/src/aurmr_tahoma/aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/'
+UOC_PATH = '/home/aurmr/workspaces/uois_soofiyan_ws/src/aurmr_tahoma/aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/'
```
[the remark this answers is not in the document]
Comment (markusgrotz) [reply aurmr_unseen_object_clustering/src/aurmr_unseen_object_clustering/tools/run_network.py:37]: See above

File: aurmr_perception/src/aurmr_perception/grasp_detection.py [hunk 4]
```diff
@@ -162,7 +163,7 @@ def clamp(self, num, min_value, max_value):
 
     def detect_grasps_cb(self, request):
         cv_image = self.bridge.imgmsg_to_cv2(request.mask, desired_encoding='passthrough')
-        cv2.imwrite("/home/aurmr/workspaces/soofiyan_ws/src/segnetv2_mask2_former/Mask_Results/grasp_pre_mask.png", cv_image)
+        cv2.imwrite("/home/aurmr/workspaces/uois_soofiyan_ws/src/segnetv2_mask2_former/Mask_Results/grasp_pre_mask.png", cv_image)
```
[the remark this answers is not in the document]
Comment (markusgrotz) [reply aurmr_perception/src/aurmr_perception/grasp_detection.py:166]: See above

**Review by markusgrotz — CHANGES_REQUESTED** [review]

**Comment by nickswalker** [comment]
Closing because it's unclear if this is still needed after #21

**Pull request closed without merging**